packages/plugin-types/src/index.ts

```diff
@@ -16,3 +16,5 @@ export * from './deployment_type.js';
 export * from './resource_access_acceptor.js';
 export * from './ssm_environment_entries_generator.js';
 export * from './package_manager_controller.js';
+export * from './printer.js';
+export * from './colors.js';
```

Comment:
Does this belong in `plugin-types` ?

Are we going to pass colors and printers to components other than cli, sandbox or create-amplify ? (should we ??)
Should we instead have `cli-types` or perhaps just have these types in `cli-core` ?

I'm worried that if we move this to `plugin-types` that's going to be invitation to put print statements in low level packages.

@edwardfoyle @Amplifiyer thoughts ?

Reply:
If the types are only needed by consumers of `cli-core` (which should be the case) then it makes sense for the types to live in that package

Reply:
Ok. I'll make another PR to use `printer` in the `cli-core` instead (so that I don't have to fix the test failure in this PR 😁 )